client/blocks/product-selector/index.jsx

```diff
@@ -130,7 +130,16 @@ export class ProductSelector extends Component {
 		const { currencyCode, intervalType, products, storeProducts } = this.props;
 
 		if ( isEmpty( storeProducts ) ) {
-			return null;
+			return map( products, product => {
+				return (
```

Comment:
Nit: We could simplify `{ return ( ... ) }` to `( ... )`. This occurs twice inside this code block.